### ISSUE TITLE
empty zones that are "asleep" will miss updates that happen onGameDay etc (the objects in the zones miss it)

### ISSUE THREAD

**Issue by TeoTwawki** [project-topaz/topaz#1384]
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
Example: enter Sacrarium and see the wrong days door/floor configuration because its running whatever the last update it saw was. we need  something to update these zones when they awaken from sleep.

**Comment by zach2good** [comment]
I was looking at these systems today, commenting so I'm reminded of this later (Zone OnCountIncrease CreateTimer, add lua binding there)

**Comment by cocosolos** [comment]
Just wanna clarify that the stuff in `onGameDay` still gets run when the zone is empty, the problem with Sacrarium is that it calls `setAnimation` which relies on the AI controller to update the entity, which is what gets killed when a zone empties.

**Comment by TeoTwawki** [comment]
> Just wanna clarify that the stuff in `onGameDay` still gets run when the zone is empty, the problem with Sacrarium is that it calls `setAnimation` which relies on the AI controller to update the entity, which is what gets killed when a zone empties.

Yeah I think before the great AI rewrite when a zone "woke up" the 1st tick was supposed to make all the AI do an immediate update with whatever happened during the zones sleep happening all immediately. No idea if that ever worked correctly, but that was what had been intended.
